CodingTracker.AhmadJer99/CodingTracker/Properties/launchSettings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "profiles": {
+    "CodingTracker": {
+      "commandName": "Project",
+      "workingDirectory": "C:\\Users\\AhmadJerjees\\source\\repos\\CodingTracker\\CodingTracker"
```

Comment:
🟠 **Hard-Coded Paths**

💡 How many machines in the world have this path? Only 1? ... 🤔 

🔧 Make sure you include this as a prerequisite step to change. Document it in the README.